about.html

```diff
@@ -0,0 +1,115 @@
+				</nav>
+			</div>
+		</div>
+    <div class="about--hero--section">
+      <section class="about--header--section">
+        <p class="about--hero--greeting">"Hello! Sharing Wolrd"</p>
+        <h1>creative commons <br> global summit 2015</h1>
+      </section>
+      <article class="about--summit--info">
+        <p>The Global Summit brings together the community of  
+          experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+          This year, we hope to expand our invitation list including organizations and individuals.</p>
+      </article>
+      <article class="about--contact--section">
+        <p class="about--contact--info">Please contact us per Email for any further questions about CC Global Summit 2015!</p>
+        <a href="mailto:emiolasaheed1@gmail.com" class="about-contact--info--mail">emiolasaheed1@gmail.com</a>
+      </article>
+    </div>
+    </header>
+    <main>
+      <section class="main--about--info">
+        <div class="main--about--header--container">
+          <h3 class="main--about--header">Creative Commons <br class="about--header--break">
+            Global Summit 2015 Logo</h3>
+          <div class="main--about--header--line"></div>
+        </div>
+        <p class="main--about--header--text">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</p>
+      </section>
+      <div class="header--logo main--about--logo">
+				<div class="logo--design about--logo--design">
+						<p>OS</p>
+				</div>
+				<p class="logo--text">OS Summit <br>2023</p>
+			</div>
+      <section  class="main--about--info">
+        <div class="main--about--header--container">
+          <h3 class="main--about--header">See the past CC Global Summits</h3>
+          <div class="main--about--header--line"></div>
+          <p class="about--see--para">Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+      </section>
+      <div class="about--image--container">
+        <div class="about-image"></div>
+        <div class="about-image"></div>
+      </div>
+      <section class="partners--section">
+        <div class="partners--header--container">
+          <h3 class="partners--header">Partners</h3>
+          <div class="partners--header--line"></div>
+        </div>
+        <div class="partners--logos--container">
+          <img src="images/mozilla 1.png" alt="Mozilla logo">
+          <img src="images/googlekorea 1.png" alt="Google logo">
+          <img src="images/naver 1.png" alt="Naver logo">
+          <img src="images/daumkakaologo 1.png" alt="Daumkakao logo">
+          <img src="images/airbnb 1.png" alt="Airbnb logo">
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="footer--logo">
+        <div class="footer--design">
+            <p>OS</p>
+        </div>
+        <p class="footer--text">Oscafest <br> commit</p>
```

Comment:
- [ ] Nice work. If you note well, you will see the original design has a dark background in the Desktop about page, and also it does not have a partner section. Let's kindly make these changes.
- [ ] kindly add the height of your two images as well, so they follow the design.
Yours
![Screenshot (1187)](https://github.com/Akorede3133/oscafest-conference/assets/65298048/00ea2a93-2f5d-4386-961d-3046a13f23fc)
provided design
![Screenshot (1159)](https://github.com/Akorede3133/oscafest-conference/assets/65298048/c82c5546-a039-442b-a851-6a3b5a5d4b29)